src/types/requestArgs.ts

```diff
@@ -227,10 +227,15 @@ export interface CreateQueryParams extends BaseCRUDParams {
 
 export enum ColumnType {
   Varchar = "varchar",
+  Varbinary = "varbinary",
+  Uint256 = "uint256",
+  Int256 = "int256",
+  Bigint = "bigint",
   Integer = "integer",
   Double = "double",
   Boolean = "boolean",
   Timestamp = "timestamp",
+  Date = "date",
 }
 
 export interface SchemaRecord {
```

Comment:
Reference to the structure declaration.
```suggestion
// https://docs.dune.com/api-reference/tables/endpoint/create#body-schema
export interface SchemaRecord {
```